src/main/webapp/script.js

```diff
@@ -31,37 +31,41 @@ async function getYouTubeComments() {
 
 /**
  * Fetches data and adds to html
+ * @param {String} url youtube url to retrieve comments
  */
-async function getChart() {
-  $('form').submit(async function() {
+async function getChart(url) {
+  $('.button').click(async function() {
+    document.getElementById('video-results').style.display = 'none';
     document.getElementById('loading-img').style.display = 'block';
-    commentStats = await getYouTubeComments();
+
+    commentStats = await getYouTubeComments(url);
```

Comment:
Does getYouTubeComments require a url parameter? It doesnt look like it from the func declaration